docs/rancher/csi-driver.md

```diff
@@ -276,3 +276,29 @@ Now you can create a new StorageClass that you intend to use in your guest Kuber
     :::
 
 1. You can now create a PVC based on this new **StorageClass**, which utilizes the **Host StorageClass** to provision volumes on the bare-metal Harvester cluster.
+
+## Upgrade the CSI Driver
+
+### Upgrade RKE2
+
+You can upgrade the CSI driver by upgrading RKE2 using the Rancher UI.
```

Comment:
Can we write description something like below
"To upgrade the CSI driver, use the Rancher UI to upgrade RKE2. Ensure the new RKE2 version supports/bundled with the updated CSI driver version."

As, simply upgrading RKE2 will not upgrade the CSI driver. WDYT?

Reply:
Agree, updated it. Thanks for the suggestion.